app/services/flat_service.rb

```diff
@@ -19,6 +19,17 @@ def delete_score(score_id)
       conn.delete("/v2/scores/#{score_id}")
     end
 
+    def create_score(title)
+      file = File.read('./xml_files/template.musicxml')
+      enc = Base64.encode64(file)
+      conn.post('/v2/scores') do |request|
+        request.body = "{ 'title': #{title},
```

Comment:
I'm not sure if this going to work, since the actual request is being stubbed but did some research and this seems to be how Faraday likes body requests to come in